handler/src/main/java/io/netty/handler/ssl/JdkSslServerContext.java

```diff
@@ -221,27 +223,28 @@ public JdkSslServerContext(File trustCertChainFile, TrustManagerFactory trustMan
         }
     }
 
-    JdkSslServerContext(X509Certificate[] trustCertChain, TrustManagerFactory trustManagerFactory,
+    JdkSslServerContext(X509Certificate[] trustCertCollection, TrustManagerFactory trustManagerFactory,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 12 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS00107)